widget/charts/src/components/BarChart/BarChart.tsx

```diff
@@ -131,9 +131,17 @@ export const BarChart = (props: BarChartPropTypes) => {
         hideTooltip();
       }
     }
+    function handleScroll() {
+      if (isMobile) {
```

Comment:
In my opinion, it should not be closed by scrolling, only where it is clicked outside the content of the tooltip, it should be closed.